lib/engine/step/g_1860/buy_cert.rb

```diff
@@ -0,0 +1,281 @@
+# frozen_string_literal: true
+
+require_relative '../base'
+require_relative '../../action/buy_company'
+require_relative '../../action/buy_shares'
+require_relative '../../action/par'
+
+module Engine
+  module Step
+    module G1860
+      class BuyCert < Base
+        attr_reader :companies, :in_auction
+
+        AUCTION_ACTIONS = %w[simple_bid pass].freeze
```

Comment:
should this be a subclass of auction?

Reply:
Do you mean auctioner? There is no Auction step class. Auctioneer would have been a force-fit as it assumes bids are for a specific company.